Headers/Foundation/NSDictionary.h

```diff
@@ -67,8 +67,9 @@ extern "C" {
 			     indent: (NSUInteger)level;
 
 #if OS_API_VERSION(MAC_OS_X_VERSION_10_6, GS_API_LATEST)
-DEFINE_BLOCK_TYPE(GSKeysAndObjectsEnumeratorBlock, void,
-  GS_GENERIC_TYPE_F(KeyT,id<NSCopying>), GS_GENERIC_TYPE(ValT), BOOL*);
+/*DEFINE_BLOCK_TYPE(GSKeysAndObjectsEnumeratorBlock, void,
```

Comment:
Was a temporary (and very quick) fix to avoid a casting hell error when building WebKit, and I kept it as a commit as a reminder that I'd have to look at it later. Should not have included it in this pull request.